examples/cms-drupal/components/avatar.js

```diff
@@ -0,0 +1,22 @@
+export default function Avatar({ author }) {
+  const name = author
+    ? author.firstName && author.lastName
+      ? `${author.firstName} ${author.lastName}`
+      : author.name
+    : null
+
+  return (
+    <>
+      {author && (
+        <div className="flex items-center">
+          <img
```

Comment:
Please use next/image (refer to some of the other updated CMS examples)!

Reply:
Sure. I actually used the same components from the other examples. Maybe they were updated after. I'll make this change.

Thanks for the review @leerob

Reply:
All done. Thank you.